electron/run.js

```diff
@@ -105,15 +105,20 @@ function onData(phase, data) {
 		document.querySelector('#phylogram').hidden = false
 		load(data)
 	} else if (phase === 'pruned-identifiers') {
-		let container = document.querySelector('#omitted-results')
+		let container = document.querySelector('#omitted-container')
+		let results = document.querySelector('#omitted-results')
 
 		let formattedNames = data.map(node => {
 			let ident = node.ident ? ` [${node.ident}]` : ''
 			return `${node.name}${ident} (${node.length})`
 		})
 
-		container.innerHTML = `<pre>${formattedNames.join('\n')}</pre>`
+		results.innerHTML = `<pre>${formattedNames.join('\n')}</pre>`
 		container.hidden = false
+		if (formattedNames.length > 0) {
+			results.innerHTML = `<pre>${formattedNames.join('\n')}</pre>`
+			container.hidden = false
```

Comment:
Either remove this container.hidden or the one on like 117

Reply:
This was caused by a bad merge. Thanks for pointing this out (line 116 also needs to be snipped)